extensions/vertx-http/dev-ui-resources/src/main/resources/dev-ui/qwc/qwc-extensions.js

```diff
@@ -58,6 +58,8 @@ export class QwcExtensions extends observeState(LitElement) {
             if(page.embed){ // we need to register with the router
                 import(page.componentRef);
                 this.routerController.addRouteForExtension(page);
+            }else if(page.includeInSubMenu){ // we need to add the link to the submenu
```

Comment:
Problem I can see when using external links as tabs is that when you click on external link, it opens it in a new window/browser tab, but in original window tab is still focused

![image](https://github.com/quarkusio/quarkus/assets/43821672/99be3372-6a73-49e4-ae89-9015589a4262)

but again, IMO graphics is all about how you feel it is right, so if you think this feels right, np!

Reply:
that was for you @phillip-kruger ^^, I forgot to comment you there.

Reply:
personally, I'd think you should programmatically focus the original tab

Reply:
Yea this is something I should fix. Let me have a look .

Reply:
This is now fixed. Basically, an external link will not be a tab item , just a link